io.github.tomluchowski.OpenDungeonsPlus.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright Opendungeons Team 2024 -->
+<component type="desktop">
+    <id>io.github.tomluchowski.OpenDungeonsPlus</id>
+    <name>OpenDungeonsPlus</name>
+    <summary>manage Creatures and Dungeons in the style of Dungeon Keeper </summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+
+    <developer_name>Thomas Luchowski</developer_name>
+    <update_contact>paul424@o2.pl</update_contact>
+
+
+    <url type="help">https://discord.com/channels/1203384193585778748/1203384194374434909</url>
```

Comment:
A homepage URL is mandatory https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#url